<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alexify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="commands">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QSlider" name="horizontalSlider">
     <property name="geometry">
      <rect>
       <x>19</x>
       <y>60</y>
       <width>301</width>
       <height>20</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>21</x>
       <y>21</y>
       <width>222</width>
       <height>28</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPushButton" name="add">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resourses.qrc">
          <normaloff>:/images/icons/Add.png</normaloff>:/images/icons/Add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>48</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="3">
         <widget class="QPushButton" name="next">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resourses.qrc">
            <normaloff>:/images/icons/Next.png</normaloff>:/images/icons/Next.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="prev">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resourses.qrc">
            <normaloff>:/images/icons/Pnrevious.png</normaloff>:/images/icons/Pnrevious.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="play">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resourses.qrc">
            <normaloff>:/images/icons/Play.png</normaloff>:/images/icons/Play.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pause">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Resourses.qrc">
            <normaloff>:/images/icons/Pause.png</normaloff>:/images/icons/Pause.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="playlists">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>181</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>179</width>
       <height>389</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QListView" name="listView"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QScrollArea" name="tracks">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>-1</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>339</width>
       <height>389</height>
      </rect>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>341</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QListView" name="playlist_view"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="current_track">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>161</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Nothing is playing&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>521</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resourses.qrc"/>
 </resources>
 <connections/>
</ui>
